Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Material/MaterialVersionUpdate.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/RTTI/TypeInfo.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/containers/map.h>
+#include <AzCore/std/string/string.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        struct MaterialVersionUpdate
+        {
+            AZ_TYPE_INFO(AZ::RPI::MaterialVersionUpdate, "{B36E7712-AED8-46AA-AFE0-01F8F884C44A}");
+
+            static void Reflect(ReflectContext* context);
+
+            struct Action
```

Comment:
This is okay for now. Just FYI, since I think we are going to have more actions in the future including running a script, I think this will eventually be replaced with an abstract base class of some kind.